Azure/sonic-buildimage
# Platform 2.0 Infra and Reboot Reason support for Dell Platforms

**- What I did**
Ported Platform 2.0 Infra and Dell S6100,Z9100,S6000 Changes from master to 201904 branch
**- How I did it**
git cherry-pick and resolve conflicts
It has 3 porting
a) Platform 2.0 base infra + Z9100,S6100 Reboot Reason
b) S6000 Directory clean up
c) S6000 Reboot Reason support
**- How to verify it**
Currently unable to verify because linux-kernel build is failing due to http timeout
**- Description for the changelog**

Platform 2.0 Infra and Reboot Reason support for Dell Platforms

**- A picture of a cute animal (not mandatory but encouraged)**
